src/pages/Write/hooks/useCompressImage.ts

```diff
@@ -0,0 +1,28 @@
+import imageCompression from 'browser-image-compression';
+
+const useCompressImage = () => {
+  const compressImage = async (file: File) => {
+    const resizingBlob = await imageCompression(file, {
+      maxSizeMB: 1,
+      maxWidthOrHeight: 750,
+      useWebWorker: true,
+    });
+    const compressedImage = new File([resizingBlob], file.name, {
+      type: file.type,
+    });
+
+    return compressedImage;
+  };
+
+  const compressImages = async (files: FileList) => {
+    const compressedImages = await Promise.all(
+      [...files].map((file) => new Promise((resolve) => resolve(compressImage(file)))),
```

Comment:
```suggestion
    const compressedImages = await Promise.all([...files].map((file) => compressImage(file)));
```
compressImage 자체가 Promise를 리턴하고 있어서, 이렇게 수정해도 괜찮지 않을까요?

Reply:
바꾸니까 타입도 제대로 설정되어 as를 쓰지 않아도 되네요
<img width="745" alt="image" src="https://github.com/ConceptBe/conceptbe-frontend/assets/33623078/227c1d8e-27ac-4335-a5fc-6fa6e398ab39">

Reply:
아 그러네요 제가 착각했군여 변경해주시고 푸쉬해주시면 될 것 같습니다 👍👍